### PR TITLE
Update uml

### LINKED ISSUE #17: Update UML diagrams
As we've started to implement our UML diagram, we have spotted some problem in them.
Now, our diagrams differ from the implementation, and they should be updated.

@Hamilcare, could you mention which parts should be particularly verified?

### PR DESCRIPTION
**Pull request by Hamilcare**
Added a new Abstract Class CommandPlayerBoat.
Update UML diagram. 
close #17

### CODE REVIEW

File: TarponSimulator2017/Controller/CommandPlayerBoat.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Tarpon.Core;
+
+namespace Tarpon.Controller
+{
+	public abstract class CommandPlayerBoat : Command
```
Comment (superboum) [inline TarponSimulator2017/Controller/CommandPlayerBoat.cs]: Nice idea, centralizing the boat management in a class.

File: TarponSimulator2017/Controller/MasterController.cs [hunk 2]
```diff
@@ -23,6 +23,7 @@ public void ChangeScene (Scene newScene)
 
 		public void Update(GameTime gameTime, KeyboardState _keyboardState, KeyboardState _oldKeyboardState, MouseState mouseState)
 		{
+//			Console.WriteLine ("update MasterController");
```
Comment (superboum) [inline TarponSimulator2017/Controller/MasterController.cs]: I think we can avoid commented debug statements in the committed code ?
To ease your tests, I recommend you the integrated debugger in monodevelop, which enable you to put breakpoints wherever you want without polluting your code 😜

**Comment by Hamilcare** [comment]
"Why have you added a IController in the UML ? (I've not checked if the class exist in our current codebase)"
My bad, didn't enough introduce my PR.
My idea was just to take a snapshot of our codebase, 
I do not know fi IController is needed, i just want it on the diagram because it does exist :)
From now on, existing classes are in yellow on diagrams, and the classes that we wrote but didn't implemant are in a light yellow.
UML diagrams should now be up to date, regardless of what is good or wrong in modelisation :)

**Comment by superboum** [comment]
> We have for instance the class "WorldBuilder" in our codebase but it's empty. What should we do on diagram

We can postpone this problem for now. When our World will be more dense, this class should become useful. At this time, we will be able to complete the code and the diagram.

> PS : should we write method signature on diagram ? Should diagram be comprehensive ?

I think it will depend on our motivation. I think that could be interesting, but at the same time I'm not motivated to maintain a perfect synchronization between the UML diagram. I think we can be realistic and update regurlarly the diagram with a maximum amount of information.

**Pull request merged**